### DISCUSSION TITLE
How to get the USB device to work without disabling SIP?

### DISCUSSION THREAD

**Discussion by fblaster** [chris1111/Wireless-USB-Big-Sur-Adapter#46] [Q&A]
Hi Chris, nice to meet you, my name is Freddy, I am writing to you from Guayaquil, Ecuador. I am writing to ask you about the possibility of activating my TP-Link Archer T4U USB device on My Macbook-Pro with Big Sur without deactivating (SIP), I do not want to have to be exposed, without any type of system security, to use my device .

I have tested version 07 of your Wireless-USB-Big-Sur-Adapter, it works perfectly, as long as SIP is disabled, I have followed all the steps and it really works, but the moment I re-enable system security, the device it is no longer detected by your software. and it is not possible to use it.

What could you do to fix it? if you need collaboration, let me know.

Best regards.

Atte.
Freddy Flores

**Comment by chris1111** [comment]
there is no way since Big Sur Beta 5
